src/pages/Home/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { ButtonNavigation, Header, Hero, GenderButton } from "../../components";
+
+const Home = () => {
+  const midCards = [
+    {
+      link: "/your-link-url",
+      imageSrc: "images/TenthSection/11.webp",
+      text: "Windows Users Didn't Know This Simple Trick To Block All Ads (Do It Now)",
+      horizontal: true,
+    },
+  ];
+  return (
+    <>
+      <Header />
+      <ButtonNavigation />
+      <Hero />
+      <GenderButton />
+    </>
+  );
+};
+
+export default Home;
```

Comment:
The Idea of pages itu ga harus simple dan dikit HTML
Component -> Reusable anywhere tanpa tau dia bakal dipake dimana
UIKit/UIComponent-> Spesifik buat pages krn attached ama spesifik UI
Pages -> Consists of 1 or more UIkit

Overall ada 2 tujuan buat component in general
1. Separation of code (mungkin ada complex function or hooks atau state)
2. Reusability
Kalo cuman UI aja, overkill buat bikin component baru
Dan bakal lebih susah di maintain krn terlalu banyak component, navigasi di kode dan kalo mau ngubah sesuatu pun jg kurang jelas dan flawles

Di case ini menurutku belom terdefine secara ok apa itu component dan apa itu UIcomponent

Imagine kalo gaada kejelasan term component, kalo misal tb tb di page X aku ngubah component Y yang kukira ini UI Component jadi spesifik ke page X, tp ternyata Y ini dipake di banyak page terus secara ga langsung krn aku gatau kejelasan ttg component, jadinya aku secara ga sengaja ngubah semuanya